### PR TITLE
Edit item

### PR DESCRIPTION
**Pull request by nabgaryan**

### CODE REVIEW

File: src/pages/todoApp/TodoApp.js [hunk 1]
```diff
@@ -6,43 +6,46 @@ import React from 'react';
...
+
   };
 
-const onCklikSave = (index, ckliked, editFieldText) =>{
-  let arr = [...listItems];
-  arr[index].value = editFieldText;
-  arr[index].ckliked = ckliked;
-  setListItems(arr);
-};
+  const onCklikSave = (index, isEditing, editFieldText) => {
```
Comment (vanahit1989) [inline src/pages/todoApp/TodoApp.js:32]: correct this name

File: src/pages/todoApp/components/listItem/ListItem.js [hunk 2]
```diff
@@ -22,50 +20,30 @@ const ListItem = ({ item, onDeleteItem, onChecked, index, onEdit, onCklikSave })
...
-                    </div>
-                    <div>
-                        <FontAwesomeIcon onClick={onEditHandler} icon="fa-solid fa-pen-to-square" />
-                        <FontAwesomeIcon onClick={onDeletHandler} icon={faTrash} />
-                    </div>
-                </SListItem>
-            </div>
-        );
-    }
+    const cnClickCancelHandler = () => {
```
Comment (vanahit1989) [inline src/pages/todoApp/components/listItem/ListItem.js:33]: pay attention on how you writing names

File: src/pages/todoApp/components/listItem/ListItem.js [hunk 3]
```diff
@@ -22,50 +20,30 @@ const ListItem = ({ item, onDeleteItem, onChecked, index, onEdit, onCklikSave })
...
                 <Button inline='inline' onClick={onClickSaveHandler} >Save</Button>
-                <Button inline='inline'>Cancel</Button>
+                <Button inline='inline' onClick={cnClickCancelHandler}>Cancel</Button>
             </div>
         );
-    }
+    } else {
 
     return (
         <div>
```
Comment (vanahit1989) [inline src/pages/todoApp/components/listItem/ListItem.js:49]: do you need div wrapper ?
i think no

File: src/pages/todoApp/TodoApp.js [hunk 4]
```diff
@@ -6,43 +6,46 @@ import React from 'react';
 
 const TodoApp = () => {
   const [listItems, setListItems] = useState([]);
-  
-   // {value: string, checked: boolean}
+
+  // {value: string, checked: boolean}
```
Comment (vanahit1989) [inline src/pages/todoApp/TodoApp.js:10]: logic is correct just remove all unnecessary comments and correct all fields

**Review by vanahit1989 — APPROVED** [review]

File: src/pages/todoApp/components/listItem/ListItem.js (hunk 3 above)
Comment (nabgaryan) [reply src/pages/todoApp/components/listItem/ListItem.js:49]: without wrapper it will give error

**Pull request merged**